ui/src/components/ripple-out/support-sections/supportContent.json

```diff
@@ -97,6 +97,10 @@
       {
         "title": "Why make a pledge?",
         "data": "By making a pledge, you play a crucial role in helping us realize our vision for a vibrant and nurturing space where we can reach out to more people with the Gospel. Your support will enable us to create an environment that fosters spiritual growth, offers a gathering place for our community, and facilitates meaningful outreach programs."
+      },
+      {
+        "title": "Can I still pledge?",
+        "data": "We have closed pledging for the campaign and we are humbled by your participation in achieving our vision to saturate Hong Kong. We extend our heartfelt gratitude to each and every one of you who have wholeheartedly pledged to be a part of the Ripple Out Campaign, and encourage you to be faithful to completing the pledge. If you still wish to participate, please visit the 'GIVE' section for more information on how you can contribute. Your support means the world to us as we strive to make a lasting impact."
```

Comment:
can you add like 'to make a lasting impact for Jesus/for the Gospel'. feels like something is missing but otherwise ok lah